packages/fast-components/src/flipper/index.ts

```diff
@@ -0,0 +1,14 @@
+import { customElement } from "@microsoft/fast-element";
```

Comment:
when do we choose "customElement" (lowercase 'c') vs "Flipper" (upper case 'F') in our naming standards?  Have we documented our naming expectations?

Reply:
These are two different things and both rooted existing standards and common practices. `@customElement` is a decorator which defines our element as a custom element - it's standards based. `Flipper` and similarly `FASTFlipper` are classes which is why they are capitalized.